packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -163,15 +163,26 @@ export const OptimizeCiemssOperation: Operation = {
 // Get the optimization result file
 // Concat the optimization result file with the optimization interventions from simulation object
 export async function getOptimizedInterventions(optimizeRunId: string) {
+	// This is a camel case version of types/Types OptimizeInterventions as we are reading from simulation execution payload
+	interface OptimizeInterventionsCamelCase {
+		intervention_type: string;
+		param_names: string[];
+		param_values?: number[];
+		start_time?: number[];
+		objective_function_option: string;
+		initial_guess?: number[];
+		relative_importance: number;
+	}
+
 	const allInterventions: Intervention[] = [];
 	// Get the interventionPolicyGroups from the simulation object.
 	// This will prevent any inconsistencies being passed via knobs or state when matching with result file.
 	const simulation = await getSimulation(optimizeRunId);
 
 	const simulationStaticInterventions: any[] = simulation?.executionPayload.fixed_static_parameter_interventions ?? [];
-	const optimizeInterventions = simulation?.executionPayload?.optimize_interventions;
+	const optimizeInterventions: OptimizeInterventionsCamelCase[] = simulation?.executionPayload?.optimize_interventions;
```

Comment:
does this not come through server to fix the camel case issue?

Reply:
As discussed this `executionPayload` is untyped 
I am doubtful we will end up typing it out as the interfaces have been updated a lot but we would likely want to :

abstract ExecutionPayload<T> with one T per simulation type
I believe the complete list is:
```
simulate
calibrate
optimize
ensemble simulate
ensemble calibrate
funman validation 
```